<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>affichagepersonnel</class>
 <widget class="QDialog" name="affichagepersonnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/afficher.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Trier Presence</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/pdf.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/delete.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherche_date">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherche_grade">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GRADE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRESENCE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SALAIRE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DATE CONTRAT</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="recherche_presence">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherche_cin">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherche_salaire">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tabpersonnel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/update.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Trier CIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/printer.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/focus.png);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:tranparent;
border-image:url(:/images/icones/back.jpg);
background:none;
border:none;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>recherche_date</zorder>
  <zorder>recherche_grade</zorder>
  <zorder>comboBox</zorder>
  <zorder>recherche_presence</zorder>
  <zorder>recherche_cin</zorder>
  <zorder>recherche_salaire</zorder>
  <zorder>tabpersonnel</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
